storage/localstore/localstore.go

```diff
@@ -362,6 +412,113 @@ func (db *DB) po(addr chunk.Address) (bin uint8) {
 	return uint8(chunk.Proximity(db.baseKey, addr))
 }
 
+func (db *DB) ListPinnedFiles() {
+	_ = db.pinFilesIndex.Iterate(func(item shed.Item) (stop bool, err error) {
+		log.Info("Pinned file", "Address", fmt.Sprintf("%0x", item.Address), "Size", item.TreeSize)
+		return false, nil
+	}, nil)
+}
+
+func (db *DB) ShowDatabaseInformation() {
```

Comment:
do you need this?

Reply:
This was one thing that is missing. I think it is extremely useful command to display the DB with its metadata. I debugged many issues with this.

Reply:
I removed this function.